vue-app/src/components/Cart.vue

```diff
@@ -289,7 +285,17 @@
           {{ tokenSymbol }}
         </div>
       </div>
-      <!-- TODO: reallocation bar -->
+      <div
+        v-if="$store.getters.isRoundJoinOnlyPhase && $stores.state.currentRound"
+        class="reallocation-section"
+      >
+        Round opens for contributing in
+        <time-left :date="$store.state.currentRound.startTime" />.
```

Comment:
Did you confirm this actually works? My misunderstanding might be off here but...
- If we're in the "join only" phase, then the round shouldn't be open for contributions
- We only know `currentRound.startTime` if a round has been deployed
- If a round has been deployed, then the round should be open for contributions

Am I correct here? If so, this logic doesn't add up.

Reply:
The copy says its ready for contributing in xxx days, so I dont think this would be open for contributions yet. Just a CTA to get BrightID verified. Maybe I am misunderstanding the isRoundJoinOnlyPhase a bit here, I thought this getter is for where a currentRound is deployed, but hasnt started yet. If there is no round, then there is a third conditional saying there is no current round which I would think covers this. Maybe we can have a quick call tomorrow to clear this up.

Reply:
From the MACI contract:
```
// When the contract was deployed. We assume that the signup period starts
// immediately upon deployment.
uint256 public signUpTimestamp;
```

Reply:
So, the `startTime` is equal to the FundingRound deploy datetime.

Reply:
So, `isRoundJoinOnlyPhase` is always going to be `false` when the `currentRound` exists.

Reply:
However, I do think that we can keep this logic but for `isRoundJoinPhase` and show a different copy. Basically to fix #413

Reply:
Thanks for clarification everyone. Mental model was a bit backwards for this, I understand now. Pushed up changes to remove this.